app.js

```diff
@@ -1,15 +1,25 @@
 const express = require('express');
 const { getTopics } = require('./controllers/topics-controllers');
+const { getArticle } = require('./controllers/articles-controllers');
 const { getApis } = require('./controllers/api-controllers');
 const app = express()
 
 app.get('/api/topics', getTopics);
 
+app.get('/api/articles/:article_id', getArticle);
 app.get('/api', getApis);
 
 app.get('/api/*', (req, res) => {
     return res.status(404).send({msg: '404: Not found'})
-})
+});
 
+app.use((err, req, res, next) => {
+    if (err.code) {
```

Comment:
This is good for now but you may want to split this up into psql and custom errors (and use next to move down the chain) in future. You can also extract it away from the app.js file when you get to the point.